src/pages/MyPage/MyPage.module.css

```
@@ -8,70 +10,27 @@
-    gap: 16px;
-    position: absolute;
-    right: 50px;
+
+.spanWrapper {
     display: flex;
+    flex-direction: column;
 }
-.imgBox {
-    padding: 16px;
+.userTitle {
 }
```

Stylelint (17.4.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)